capstone/src/main/java/com/google/sps/service/FilterService.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.service;
+
+import java.io.IOException;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory.Builder;
```

Comment:
Usually for the builder patterns you dont import the full path because that would mean if you also wanted to use something like Query.Builder they would conflict. So you just import up to the base class (KeyFactory) and then when you reference it you can use Key key = new KeyFactory.Builder(...)...